src/background/background.js

```
@@ -51,33 +52,58 @@
 // Listen for tab updates to switch popups
 chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (changeInfo.url) {
-    if (tab.url.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
+    if (tab?.url?.includes('bigmachines.com/admin/ui/branding/edit_header_footer.jsp')) {
       chrome.action.setPopup({
         tabId: tabId,
         popup: 'popup/popupHeaderFooterHTML.html'
       });
-    } else if (tab.url.includes('bigmachines.com')) {
+    } else if (tab?.url && tab.url.includes('bigmachines.com/admin/interfaceCatalogs/list_ics_resources.jsp')){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)